### PR TITLE
Force using posix paths

### PR DESCRIPTION
**Pull request by suricactus**
So even if the user have set `'DCIM\\IMG_' + $id + '.jpg'`, we store it correctly.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f361154b0c220ffef87692e03f2f33511959dea4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-d2d873aed618d9f6414834f11eb2a6bf2d8464f0-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-d2d873aed618d9f6414834f11eb2a6bf2d8464f0-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-d2d873aed618d9f6414834f11eb2a6bf2d8464f0-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-d2d873aed618d9f6414834f11eb2a6bf2d8464f0-x86.apk)

File: src/qml/editorwidgets/ExternalResource.qml [hunk 1]
```diff
@@ -114,12 +114,12 @@ EditorWidgetBase {
 
   function getPictureFilePath() {
     var evaluatedFilepath = expressionEvaluator.evaluate()
+    var filepath = ( evaluatedFilepath && FileUtils.fileSuffix(evaluatedFilepath) !== '' )
+      ? evaluatedFilepath
+      : ('DCIM/JPEG_' + (new Date()).toISOString().replace(/[^0-9]/g, '') + '.jpg')
+    filepath = filepath.replace('\\', '\/')
```
Comment (m-kuhn) [inline src/qml/editorwidgets/ExternalResource.qml:120]:
```suggestion
    filepath = filepath.replace('\\', '/')
```

no? you're the js expert

**Review by m-kuhn — APPROVED** [review]
+1000

File: src/qml/editorwidgets/ExternalResource.qml (hunk 1 above)
Comment (suricactus) [reply src/qml/editorwidgets/ExternalResource.qml:120]: both are fine, in js `\` char in strings escapes whatever is afterwards.

Comment (suricactus) [reply src/qml/editorwidgets/ExternalResource.qml]: I removed the escaping char anyways.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 609369e7fbcd9b542a969c1485c182d4548b7a13:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-7d00bf7ced1361b447829be6cdd5b8f183f6f567-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-7d00bf7ced1361b447829be6cdd5b8f183f6f567-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-7d00bf7ced1361b447829be6cdd5b8f183f6f567-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2071-7d00bf7ced1361b447829be6cdd5b8f183f6f567-x86.apk)

**Pull request merged**